### PR TITLE
Issue #258: Allow connection tests for the APP administrator

### PR DESCRIPTION
**Pull request by GoebelL**
The ping function has been refactored to work for the APP administrator without technolgy manager role. The controllers theirselves are using the platform service to load their settings without exposing their credentials.

---
This change is [Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)

### CODE REVIEW

**Review by kowalczyka — CHANGES_REQUESTED** [review]
> Reviewed 2 of 21 files at r1, 3 of 8 files at r2.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 5 of 21 files reviewed, 1 unresolved discussion (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

___
*[oscm-app-openstack/javasrc/org/oscm/app/openstack/controller/OpenStackController.java, line 714 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfJ9vG22_dKPqEy7osw:-LfJ9vG22_dKPqEy7osx:beniki7) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-openstack/javasrc/org/oscm/app/openstack/controller/OpenStackController.java#L714)):*
> ```Java
> 	 * is unable to obtain or is incomplete
> 	 */
> 	private HashMap<String, Setting> getControllerSettings() throws ConfigurationException {
> ```

Is this method used anywhere?

**Review by kowalczyka — CHANGES_REQUESTED** [review]
> Reviewed 8 of 21 files at r1, 5 of 8 files at r2.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 18 of 21 files reviewed, 2 unresolved discussions (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

___
*[oscm-extsvc-notification/javares/META-INF/maven/org.oscm/oscm-extsvc-notification/pom.properties, line 1 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfJBMXABPbA_qQ5lbUV:-LfJBMXABPbA_qQ5lbUW:b-rdw93b) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-extsvc-notification/javares/META-INF/maven/org.oscm/oscm-extsvc-notification/pom.properties#L1)):*
> ```INI
> #Generated by Maven
> ```

Do we need this file commited to the repo?

**Review by kowalczyka — COMMENTED** [review]
> Reviewed 1 of 21 files at r1.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 19 of 21 files reviewed, 2 unresolved discussions (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

File: oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties [hunk 1]
```diff
@@ -13,7 +13,7 @@ error_db_save_conf=Fehler beim Speichern der Controllereinstellungen.
 error_db_seq=Fehler beim Zugriff auf die Datenbanksequenz {0}.
 error_db_validate_ip_not_found=Fehler bei der Validierung der IP-Addresse {0}. Sie wurde im VLAN {1} nicht gefunden.
 error_db_validate_ip_unknown_cluster=Fehler bei der Validierung der IP-Addresse. Cluster {0} unbekannt.
-error_db_validate_ip_unknown_vlan=Fehler bei der Validierung der IP-Addresse. VLAN {0} unbekannt.�
+error_db_validate_ip_unknown_vlan=Fehler bei der Validierung der IP-Addresse. VLAN {0} unbekannt.ó
```
Comment (Crystalzord) [inline oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties]: I believe that this "ó" at the end of line is there by mistake.

**Review by GoebelL — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 19 of 20 files reviewed, 3 unresolved discussions (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

___
*[oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties, line 16 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-Lf4DtrIwRbUZzKq4YGY-r2-16:-LfJLpiHCZP4Sxuvyc49:bffjvkx) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties#L16)):*
<details><summary><i>Previously, Crystalzord (Lukasz Wiecek) wrote…</i></summary><blockquote>

I believe that this "ó" at the end of line is there by mistake.
</blockquote></details>

It's not from my change. But I believe the same, so I will check and remove those message flows.
___
*[oscm-extsvc-notification/javares/META-INF/maven/org.oscm/oscm-extsvc-notification/pom.properties, line 1 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfJBMXABPbA_qQ5lbUV:-LfJMl9oBIZqqrSpUt13:bg7srta) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-extsvc-notification/javares/META-INF/maven/org.oscm/oscm-extsvc-notification/pom.properties#L1)):*
<details><summary><i>Previously, kowalczyka (Arkadiusz Kowalczyk) wrote…</i></summary><blockquote>

Do we need this file commited to the repo?
</blockquote></details>

Not needed here and removed.

**Review by GoebelL — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 18 of 20 files reviewed, 4 unresolved discussions (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

___
*[oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties, line 16 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfK6d0DBIciv41XKMN4:-LfK6d0DBIciv41XKMN5:bb6p9ni) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties#L16)):*
> ó
___
*[oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties, line 16 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-Lf4DtrIwRbUZzKq4YGY-r2-16:-LfK7aWE01E-RBXBccWo:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties#L16)):*
<details><summary><i>Previously, GoebelL (Lorenz Goebel) wrote…</i></summary><blockquote>

It's not from my change. But I believe the same, so I will check and remove those message flows.
</blockquote></details>

Done.

**Review by kowalczyka — COMMENTED** [review]
> Reviewed 1 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 18 of 20 files reviewed, 3 unresolved discussions (waiting on @Crystalzord, @GoebelL, and @kowalczyka)

**Review by kowalczyka — COMMENTED** [review]
> Reviewed 1 of 21 files at r1, 1 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: all files reviewed, 3 unresolved discussions (waiting on @Crystalzord and @GoebelL)

___
*[oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties, line 16 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfK6d0DBIciv41XKMN4:-LfK94U53W7BK7Jn74ci:bb6ywmx) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-vmware/javares/org/oscm/app/vmware/i18n/messages_de.properties#L16)):*
<details><summary><i>Previously, GoebelL (Lorenz Goebel) wrote…</i></summary><blockquote>

> ó
</blockquote></details>

LGTM

**Review by GoebelL — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: 19 of 20 files reviewed, 2 unresolved discussions (waiting on @Crystalzord and @kowalczyka)

___
*[oscm-app-openstack/javasrc/org/oscm/app/openstack/controller/OpenStackController.java, line 714 at r2](https://reviewable.io/reviews/servicecatalog/oscm/308#-LfJ9vG22_dKPqEy7osw:-LfK9vJh1XebqV8Jc5c9:b-9mbr0t) ([raw file](https://github.com/servicecatalog/oscm/blob/5a56553607029d8de2df87c71ec1b7539a3f2fcb/oscm-app-openstack/javasrc/org/oscm/app/openstack/controller/OpenStackController.java#L714)):*
<details><summary><i>Previously, kowalczyka (Arkadiusz Kowalczyk) wrote…</i></summary><blockquote>

Is this method used anywhere?
</blockquote></details>

Nope, I removed it. Well noticed :-)

**Review by kowalczyka — APPROVED** [review]
> Reviewed 1 of 21 files at r1, 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: all files reviewed, 1 unresolved discussion (waiting on @Crystalzord)

**Review by GoebelL — COMMENTED** [review]
:lgtm:

> Reviewed 11 of 21 files at r1, 7 of 8 files at r2, 1 of 2 files at r3, 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308#-:-LfKAxV92fa9AD1m4JVy:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @Crystalzord)

**Review by kowalczyka — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: all files reviewed, 1 unresolved discussion (waiting on @Crystalzord)

**Review by GoebelL — COMMENTED** [review]
> Dismissed @Crystalzord from a discussion.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/308)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**